lib.js

```diff
@@ -432,7 +432,7 @@ class sender {
    */
   setChannel(channel, value) {
     if ((channel > 511) || (channel < 0)) {
-      throw new Error('Channel must be between 0 and 512');
+      throw new Error('Channel must be between 0 and 511');
```

Comment:
If we touch this error message, we should clearly identify if 511 is included or excluded.
It is not exactly obvious from my perspective.
e.g. `0<=x<=511`